extensions/google-drive/src/index.tsx

```diff
@@ -48,7 +48,9 @@ const Command = () => {
   const [filesIndexGeneratedAt, setFilesIndexGeneratedAt] = useState<Date | null>(null);
   const db = useDb();
 
-  useEffect(initialSetup, []);
+  useEffect(() => {
+    initialSetup();
+  }, []);
```

Comment:
Can revert this back to how it was before.

Reply:
Resolved in ec741b6

Reply:
I don't see `clearLeastAccessedFilePreviewsCache()` as an async function either which is run from `initialSetup()` so no point in having async...await stuff here.